xml/admin_monitoring_alerting.xml

```diff
@@ -366,36 +366,127 @@ placement:
     </para>
   </sect1>
 
-  <!--  <sect1>
+  <sect1 xml:id="snmp-trap-receiver">
```

Comment:
The name is wrong, we are talking about a Prometheus Alertmanager receiver which translates incoming notifications into SNMP traps. A trap receiver is the counterpart which receives the SNMP traps that the software submits which we are talking about here.
```suggestion
  <sect1 xml:id="prmetheus-webhook-snmp">
```